src/lib/room.js

```diff
@@ -200,8 +204,9 @@ export const loadMessages = async () => {
 		const lastMessage = messages[messages.length - 1];
 		await store.setState({ lastReadMessageId: lastMessage && lastMessage._id });
 
-		// TODO: create a separate event for starting the call and checking if the call is ongoing
-		if (lastMessage.t === constants.webrtcCallStartedMessageType || lastMessage.t === constants.jitsiCallStartedMessageType) {
+		if (lastMessage.endTs && (lastMessage.t === constants.webrtcCallStartedMessageType || lastMessage.t === constants.jitsiCallStartedMessageType)) {
```

Comment:
```suggestion
if (lastMessage.t === constants.webrtcCallStartedMessageType || lastMessage.t === constants.jitsiCallStartedMessageType) {
			if (lastMessage.endTs) {
				await store.setState({ ongoingCall: { callStatus: 'ended', time: lastMessage.ts }, incomingCallAlert: null });
				return;
			}
			await processCallMessage(lastMessage);
		}
```

There seems to be a common condition here, so merging them makes more sense